torch/fx/passes/runtime_assert.py

```diff
@@ -145,6 +147,24 @@ def _is_intermediate_tensor_sym_call(node: fx.Node) -> bool:
             )
         )
 
+    # Figure out if we're in Dynamo or AOTAutograd
+    val_key = "val" if export else "example_value"
```

Comment:
Not sure what the best way is to figure out which key to use. Maybe we could iterate through nodes and check what's present? And then fallback to this flag if that's empty for some reason.

Reply:
I usually just make a helper that fetches either val or example_value, depending on which one is set, preferring val. There's one in symbolic_shapes